lib/upgrade_format_task.rb

```diff
@@ -44,7 +44,9 @@ def upgrade(store, name, options = {})
 
     if !errors.empty?
       Machinery.logger.error errors.join("\n")
-      raise Machinery::Errors::UpgradeFailed.new(errors.join("\n"))
+      exception = Machinery::Errors::UpgradeFailed.new(errors.join("\n"))
+      exception.hint_proc = -> { Hint.upgrade_format_force(name: name || "--all") }
```

Comment:
I don't really like changing the behavior of the hint methods that fundamentally depending on an option. What about

`exception = Machinery::Errors::UpgradeFailed.new(errors.join("\n") + Hint.to_string(:upgrade_format_force, name: name)`

instead?